### DISCUSSION TITLE
p89 SECION-036 リスト9.2の誤植

### DISCUSSION THREAD

**Discussion by YuyaAbo** [budougumi0617/go_todo_app#22] [Ideas]
タイトル通りですが、p89 SECTION-036「状態を持つ関数を作る」にあるリスト9.1は誤植でリスト9.2が正しそうです（リスト番号のみの誤植）。

**Comment by budougumi0617** [comment]
ご指摘ありがとうございます！
ちょうど正誤表を作成していたのでご指摘を追加させていただきました。もし第3刷が発行される際は出版社にご指摘内容の修正依頼をいたします。

https://github.com/budougumi0617/go_todo_app/blob/290cb5f6729d479ae571165d8114abce56f64f17/errata.md
